src/Utils/functions.jsx

```diff
@@ -267,22 +277,33 @@ export function DeleteClassificationField(
           study_pk,
           experiment_pk,
           classificationName,
-          id: values[index].id,
+          id: Array.isArray(values) ? values[index].id : values.id,
```

Comment:
Its better to remove logic from the function parameters list, put it in the bod of the function